CONTRIBUTING.md

```diff
@@ -48,6 +57,8 @@ Once you have built your template file or files, the commit message should have
 
 If you want to describe your changes in more detail, add an empty line followed by those details ([example](https://github.com/void-linux/void-packages/commit/f1c45a502086ba1952f23ace9084a870ce437bc6)).
 
+Once you've made changes to your forked repository, you can [start a pull request](#starting-a-pull-request).
```

Comment:
This line is out of order and unnecessary, changes should be tested before submitting, as stated below. I see that this relates to following paragraph, but maybe reword that instead.